src/components/form/signup.js

```diff
@@ -129,6 +129,7 @@ class Signup extends Component {
         const affiliate_token = Cookies.getJSON('affiliate_tracking')
         const signup_device = LocalStore.get('signup_device')
         const date_first_contact = LocalStore.get('date_first_contact')
+        const gclid = LocalStore.get('gclid')
```

Comment:
where are you storing this? when will you remove it?

Reply:
in traffic source, I set gclid based on url param